### PR TITLE
Version 2 corrections

### PR DESCRIPTION
**Pull request by olyzeus**
Documented with corresponding tags

### CODE REVIEW

File: contracts/BondTeller.sol [hunk 1]
```diff
@@ -101,18 +103,28 @@ contract BondTeller {
         uint256 _expires,
         address _feo
     ) external onlyDepository returns (uint256 index_) {
-        treasury.mint(address(this), _payout.add(feReward));
+        uint reward = _payout.mul(feReward).div(10_000);
```
Comment (lolchocotaco) [inline contracts/BondTeller.sol:106]: Don't think you meant to put an underscore here.

File: contracts/sOlympusERC20.sol [hunk 2]
```diff
@@ -53,7 +53,7 @@ contract sOlympus is IsOHM, ERC20Permit {
     Rebase[] public rebases; // past rebase data
 
     uint256 private constant MAX_UINT256 = ~uint256(0);
-    uint256 private constant INITIAL_FRAGMENTS_SUPPLY = 5000000 * 10**9;
+    uint256 private constant INITIAL_FRAGMENTS_SUPPLY = 5_000_000 * 10**9;
```
Comment (lolchocotaco) [inline contracts/sOlympusERC20.sol:56]: honestly a weirdone.

File: contracts/BondTeller.sol (hunk 1 above)
Comment (lolchocotaco) [reply contracts/BondTeller.sol:106]: nvm, maybe you did. i dont like this codeStyle 😢

File: contracts/StakingDistributor.sol [hunk 3]
```diff
@@ -45,11 +48,11 @@ contract Distributor is Governable, Guardable {
...
     ) {
-        require(_treasury != address(0));
+        require(_treasury != address(0), "Zero address found");
         treasury = ITreasury(_treasury);
-        require(_ohm != address(0));
-        OHM = IERC20(_ohm);
-        require(_staking != address(0));
+        require(_ohm != address(0), "Zero address found");
+        ohm = IERC20(_ohm);
+        require(_staking != address(0), "Zero address found");
```
Comment (lolchocotaco) [inline contracts/StakingDistributor.sol:55]: We should probably be more descriptive here with which param.

File: contracts/governance/gOHM.sol [hunk 4]
```diff
@@ -124,13 +124,13 @@ contract gOHM is IgOHM {
...
+        require(_staking != approved, "Invalid argument");
 
-        require(_staking != address(0));
+        require(_staking != address(0), "Zero address found");
         approved = _staking;
 
-        require(address(sOHM) == address(0));
-        require(_sOHM != address(0));
+        require(address(sOHM) == address(0), "Cannot migrate twice");
+        require(_sOHM != address(0), "Zero address found");
```
Comment (lolchocotaco) [inline contracts/governance/gOHM.sol:133]: We should be explicit about where the zero-address was found.

Comment (olyzeus) [reply contracts/governance/gOHM.sol:133]: formatted as "Zero address: parameter"

**Review by JExtor — COMMENTED** [review]
looks good

**Review by lolchocotaco — APPROVED** [review]
Look magnificent so far. Will merge this to keep the ball rolling.

**Pull request merged**